Makefile

```diff
@@ -26,55 +26,51 @@ GIT_HEAD_COMMIT_LONG  := $(shell git log -1 --format='%H')
 GIT_HEAD_COMMIT_SHORT := $(shell git rev-parse --short HEAD)
 GIT_HEAD_ABBREV       := $(shell git rev-parse --abbrev-ref HEAD)
 
-GORELEASER_TAG       ?= $(shell git describe --tags --abbrev=0)
-GORELEASER_IS_PREREL ?= $(shell $(ROOT_DIR)/script/is_prerelease.sh "$(GORELEASER_TAG)")
-
-# BUILD_TAGS are for builds withing this makefile
-# GORELEASER_BUILD_TAGS are for goreleaser only
-# Setting mainnet flag based on env value
-# export MAINNET=true to set build tag mainnet
-ifeq ($(MAINNET),true)
-	BUILD_MAINNET=mainnet
-	BUILD_TAGS=osusergo,netgo,ledger,mainnet,static_build
-	GORELEASER_BUILD_TAGS=$(BUILD_TAGS)
-else
-	BUILD_TAGS=osusergo,netgo,ledger,static_build
-	GORELEASER_BUILD_TAGS=$(BUILD_TAGS),testnet
-endif
-
-ifeq ($(GORELEASER_IS_PREREL),false)
-	GORELEASER_HOMEBREW_NAME=akash
-	GORELEASER_HOMEBREW_CUSTOM=
+RELEASE_TAG           ?= $(shell git describe --tags --abbrev=0)
+IS_PREREL             := $(shell $(ROOT_DIR)/script/is_prerelease.sh "$(RELEASE_TAG)")
+IS_MAINNET            := $(shell $(ROOT_DIR)/script/mainnet-from-tag.sh "$(RELEASE_TAG)")
+
+GO_LINKMODE            ?= external
+GO_MOD                 ?= readonly
+BUILD_TAGS             ?= osusergo,netgo,ledger,static_build
+GORELEASER_STRIP_FLAGS ?=
```

Comment:
Does this line do anything?

Reply:
yeah, it is set to `-s -w` if `BUILD_OPTIONS` does not contain `nostrip` (line 66)
in other words it is set to `GORELEASER_STRIP_FLAGS='-w -d'` unless `BUILD_OPTIONS=nostrip` used to call make target